docs/source/development.rst

```diff
@@ -0,0 +1,104 @@
+.. _development-policy:
+
+==================
+Development policy
+==================
+
+In this section, we describe the development policy the core team of Chainer Chemistry follows.
+Developers who are thinking to send PRs to the repository are encouraged to read the following sections
+before starting implementation.
+
+
+Versioning policy
+=================
+
+Basically, we follow the `semantic versioning v2.0.0 <https://semver.org>`_.
```

Comment:
If we want to fix the version of semantic versioning policy to 2.0.0, following URL may be better.
https://semver.org/spec/v2.0.0.html

Reply:
fix